src/lib/helpers.ts

```diff
@@ -33,25 +33,29 @@ export const rpcEndpointUrl = (chainId: number): string => {
   const envVarName = `RPC_URL_${chainId}`;
   const envVarValue = process.env[envVarName];
 
-  // no RPC URL provided in env var
-  if (!envVarValue) {
+  try {
+    return parseRpcEndpointUrl(envVarValue);
+  } catch (e: any) {
+    throw new Error(`Environment variable parse error: '${envVarName}' was ${e.message}`);
+  }
+};
+
+export const parseRpcEndpointUrl = (rawValue?: string): string => {
+  // no RPC URL provided
+  if (!rawValue) {
```

Comment:
@lightwalker-eth here's the split of functionality creating RPC config values. One is to read raw value from env vars, the other is to have it parsed into expected value range. We unit test the later.